kbs/docs/config.md

```diff
@@ -32,6 +31,19 @@ section:
 | `certificate`            | String       | Path to a certificate file to be used for HTTPS.                                                           | No       | -                    |
 | `auth_public_key`        | String       | Path to a public key file to be used for authenticating the resource registration endpoint token (JWT).    | No       | -                    |
 
+### Attestation Token Configuration
+
+The following properties can be set under the `attestation_token_config` section.
+
+>This section is available only when the `resource` feature is enabled.
+
+| Property                   | Type          | Description                                         | Required | Default   |
+|----------------------------|---------------|-----------------------------------------------------|----------|-----------|
+| `attestation_token_config` | String        | Attestation token broker type. Valid values: `CoCo` | Yes      | -         |
+| `trust_certs_paths`        | String Array  | Trusted root certificates file paths (PEM format).  | No       | -         |
+
+If `trust_certs_paths` is set, KBS will forcibly check the validity of the Attestation Token signature public key certificate.
```

Comment:
What if it isn't set? It seems like that will cause all requests to a remote AS to fail?

Reply:
If not set this field, KBS will skip the verification of the certificate. I have add this explain to `config.md`